packages/google-closure-compiler/lib/grunt/index.js

```diff
@@ -218,6 +224,42 @@ module.exports = (grunt, pluginOptions) => {
       });
   }
 
+  // 
+  /**
+   * Grabs `ps` as array of promise-returning functions and `done` function as callback.
+   * Separates ps` into batches of length == compileInBatches and runs resulting 
+   * promises in batches in parallel but batches in series.
+   * 
+   * @param {Array<Function(...?):Promise>}} functions returning promises
```

Comment:
```suggestion
   * @param {!Array<function():!Promise<undefined>}} ps functions returning promises
```

Reply:
Made change to closing bracket - there is > instead of } by logic if it is array of promise<undefined> returning functions